File: exercises/guessing-game.rb
secret_word = 'javascript'
guess = ''
guess_count = 0
guess_limit = 3
out_of_guesses = false
print 'Whats js short for? (3 tries): '


while guess != secret_word and !out_of_guesses
  if guess_count < guess_limit
    guess = gets.chomp
    if guess != secret_word
      print 'wrong guess, try again: '
    end
    guess_count += 1
  else
    out_of_guesses = true
  end
end
if guess == secret_word
  puts 'you won'
else
  puts 'you lose'
end

File: exercises-js/guessing-game.js
const secretWord = 'ruby';
let guess = '';
let guessCount = 1;
let guessLimit = 3;
let outOfGuesses = false;
guess = prompt('whats rb short for? ')

while (guess !== secretWord && !outOfGuesses){
  if (guessCount < guessLimit){
    if(guess !== secretWord){
      guess = prompt ('Wrong answer, try again')
    }
    guessCount ++

  }else{
    outOfGuesses = true
  }
}

if(guess === secretWord){
  alert('you won')
}else{
  alert('you lost sorry')
}
